library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -98,32 +98,19 @@ internal class TransactionPayloadFragment :
     }
 
     @SuppressLint("NewApi")
+    @Suppress("ComplexMethod")
```

Comment:
Didn't get why it is here. I tried to locally run ktlint and detekt on your branch without this annotation and everything seems fine.

Reply:
Yup I forgot to remove this